packages/maskbook/src/plugins/UnlockProtocol/SNSAdaptor/LockInList.tsx

```diff
@@ -0,0 +1,62 @@
+import { makeStyles, Theme, ListItem, ListItemText, Checkbox } from '@material-ui/core'
+import classNames from 'classnames'
+import { ChangeEvent, useCallback } from 'react'
+import type { DefaultComponentProps } from '@material-ui/core/OverridableComponent'
+import type { CheckboxProps } from '@material-ui/core/Checkbox'
+import type { ListItemTypeMap } from '@material-ui/core/ListItem'
+import type { UnlockLocks } from '../types'
+import { useStylesExtends } from '@masknet/shared'
+
+const useStyle = makeStyles((theme: Theme) => ({
+    root: {
+        cursor: 'pointer',
+        paddingLeft: 8,
+    },
+    overflow: {
+        textOverflow: 'ellipsis',
+        whiteSpace: 'nowrap',
+        overflow: 'hidden',
+    },
+    hightlighted: {
+        backgroundColor: 'inherit',
+        color: 'inherit',
+        fontWeight: 'bold',
+    },
+}))
+
+export interface LockInListProps extends withClasses<never> {
```

Comment:
```suggestion
export interface LockInListProps {
```

Since you don't have any `classes` to extends with